### PR TITLE
Support T1 to be placed outside of electrophysiological dataset

### PR DESCRIPTION
**Pull request by hoechenberger**

### CODE REVIEW

File: config.py [hunk 1]
```diff
@@ -996,6 +995,66 @@
...
+
+- the subject ID, `BIDSPath.subject`
+- the experimental session, `BIDSPath.session`
+- the BIDS root, `BIDSPath.root`
+
+This `BIDSPath` can then be modified – or an entirely new `BIDSPath` can be
+generated – and returned by the function, pointing to the T1-weighted image.
+
+Note: Note
+    The function accept and return a single `BIDSPath`.
```
Comment (agramfort) [inline config.py:1033]:
```suggestion
    The function accepts and returns a single `BIDSPath`.
```

File: config.py [hunk 2]
```diff
@@ -996,6 +995,66 @@
...
+
+???+ example "Example"
+    The MRI session is different than the electrophysiological session:
+    ```python
+    def electrophys_mri_map(bids_path):
```
Comment (agramfort) [inline config.py:1038]:
```suggestion
    def get_t1_from_meeg(bids_path):
```

File: config.py [hunk 3]
```diff
@@ -996,6 +995,66 @@
...
+        bids_path.session = 'MRI'
+        return bids_path
+
+
+    mri_t1_path_generator = electrophys_mri_map
```
Comment (agramfort) [inline config.py:1043]:
```suggestion
    mri_t1_path_generator = get_t1_from_meeg
```

File: config.py [hunk 4]
```diff
@@ -996,6 +995,66 @@
...
+    ```
+
+    The MRI recording is stored in a different BIDS dataset than the
+    electrophysiological data:
+    ```python
+    def electrophys_mri_map(bids_path):
```
Comment (agramfort) [inline config.py:1049]:
```suggestion
    def get_t1_from_meeg(bids_path):
```

File: config.py [hunk 5]
```diff
@@ -996,6 +995,66 @@
...
+        bids_path.root = '/data/mri'
+        return bids_path
+
+
+    mri_t1_path_generator = electrophys_mri_map
```
Comment (agramfort) [inline config.py:1054]:
```suggestion
    mri_t1_path_generator = get_t1_from_meeg
```

**Review by agramfort — APPROVED** [review]

**Comment by hoechenberger** [comment]
ERP CORE CI issue seems unrelated

**Pull request merged**